airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresSpecTest.java

```diff
@@ -112,4 +122,12 @@ void testWithReplicationMethodWithReplicationSlotWithNull() {
     assertFalse(validator.test(schema, config));
   }
 
+  @Test
```

Comment:
Note that this test is essentially a copy paste, would be nice to not have to deal with this someday from: https://github.com/airbytehq/airbyte/issues/10135

Reply:
https://gist.github.com/octocat/9257657?permalink_comment_id=2680808#gistcomment-2680808

Reply:
https://gist.github.com/octocat/9257657?permalink_comment_id=2680808#gistcomment-2680808